Yank/multistate/multistatesampler.py

```diff
@@ -1447,12 +1451,13 @@ def _offline_analysis(self):
         # Raise an exception after 6 times MBAR gave an error.
         if bump_error_counter:
             self._online_error_trap_counter += 1
-            if self._online_error_trap_counter >= 6:
+            # Will never be true, but code left in place in case we change logic to allow again
+            if self._online_error_trap_counter >= np.inf:
```

Comment:
What if we add a YAML option that governs how we treat errors like this? We can come up with a proposal for that in an issue and leave that implementation to a future release.

Reply:
I can make this an option, It would be pretty strait forward

Reply:
Shoot, sorry--my mistake. Can you fix the milestones for the tagged issues then?